cardigan/stories/components/MessageBar.js

```diff
@@ -0,0 +1,18 @@
+import { storiesOf } from '@storybook/react';
+import MessageBar from '../../../common/views/components/MessageBar/MessageBar';
+import Readme from '../../../common/views/components/MessageBar/README.md';
+
+const stories = storiesOf('Components', module);
+
+stories.add(
+  'BetaBar',
```

Comment:
I'd make this 'MessageBar' and remove the `BetaBar.js` story altogether (since it just implements this). Do you think that makes sense?

Reply:
The purpose is there though, and different, and what makes BetaBar a sub-component in terms of functionality?

Reply:
If BetaBar wasn't a component, it should just be implemented on the `page`, but as it is, and shared, it feels like creating the exception needs a reason?

Reply:
Hmmmm, yeah, you're probably right.